src/transform/mark.js

```diff
@@ -81,8 +81,6 @@ Step.define("removeMark", {
     let slice = doc.slice(step.from, step.to)
     slice.content = mapFragment(slice.content, node => {
       return node.mark(step.param.removeFromSet(node.marks))
-      if (!parent.type.canContainMark(step.param.type)) return node
```

Comment:
It could come from `mapFragment`, but on closer reading, we don't need the test here and that was just a copy/paste leftover.